### PR TITLE
Make CI publish `latest` tagged web-viewer to `app.rerun.io`

### PR DESCRIPTION
**Pull request by emilk**
I think the only way to test this is to merge it and then make a new releae

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -321,6 +321,14 @@ jobs:
           destination: "rerun-web-viewer/version/${{github.ref_name}}"
           parent: false
 
+      - name: "Upload web-viewer (latest release)"
+        if: github.ref == 'latest'
+        uses: google-github-actions/upload-cloud-storage@v1
+        with:
+          path: "web_viewer"
+          destination: "rerun-web-viewer/web_viewer"
```
Comment (jleibs) [inline .github/workflows/rust.yml:329]: Maybe name this “rerun-web-viewer/latest”?

File: .github/workflows/python.yml [hunk 2]
```diff
@@ -312,6 +312,14 @@ jobs:
           destination: "rerun-example-rrd/version/${{github.ref_name}}"
           parent: false
 
+      - name: "Upload web-viewer (latest release)"
+        if: github.ref == 'latest'
+        uses: google-github-actions/upload-cloud-storage@v1
+        with:
+          path: "rrd"
+          destination: "rerun-example-rrd/web_viewer"
```
Comment (jleibs) [inline .github/workflows/python.yml:320]: Maybe name this “example-rrd/latest”?

File: .github/workflows/rust.yml (hunk 1 above)
Comment (emilk) [inline .github/workflows/rust.yml:329]: I'm no longer sure this is right.

Where does <https://app.rerun.io/> even serve from? O.o

**Review by jleibs — COMMENTED** [review]
I’m not positive this will always do what we expect. I think it will end up sensitive to how the `latest` tag gets updated or created, but worth a shot.

File: .github/workflows/rust.yml (hunk 1 above)
Comment (jleibs) [reply .github/workflows/rust.yml:329]: We’ll want to modify the base rewrite rule to send app.rerun.io to whatever prefix we use for this target.

File: .github/workflows/python.yml (hunk 2 above)
Comment (emilk) [reply .github/workflows/python.yml:320]: There was already a folder in the bucket named `web_viewer` which is what I assumed `https://app.rerun.io/` was pointing to, but now I don't know where `https://app.rerun.io/` is pointing

File: .github/workflows/python.yml [hunk 3]
```diff
@@ -312,6 +312,14 @@ jobs:
           destination: "rerun-example-rrd/version/${{github.ref_name}}"
           parent: false
 
+      - name: "Upload web-viewer (latest release)"
```
Comment (teh-cmc) [inline .github/workflows/python.yml:315]:
```suggestion
      - name: "Upload RRD (latest release)"

**Review by teh-cmc — APPROVED** [review]

**Pull request merged**